css/media.css

```diff
@@ -0,0 +1,49 @@
+/* Mobile */
+@media (min-width: 500px) {
```

Comment:
I'd say these rules should reside in their stylesheets -- concers will be separated that way ;)